### PR TITLE
PhantomJs full support

### PR DESCRIPTION
**Pull request by Klemek**
Support for PhantomJs on any OS (had to add optional subpath in cache.py because binary is not at archive root)

### CODE REVIEW

**Comment by Klemek** [comment]
Tests are failing because some firefox tests are failing + 1 because PhantomJs specific 2.0 is available on windows but not on linux

**Comment by SergeyPirogov** [comment]
@Klemek so why we can't skip the test for 2.0 and Linux?

File: tests/test_phantomjs_manager.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+            if f.startswith("phantomjs-"):
+                try:
+                    os.remove(os.path.join(path, f))
+                except:
+                    pass
+
+
+
+def test_phantomjs_manager_with_specific_version():
+    bin = PhantomJsDriverManager("2.0.0").install()
```
Comment (Klemek) [inline tests/test_phantomjs_manager.py]: `bin = PhantomJsDriverManager("2.0.0", os_type="win32").install()`
would be appropriate for this test to be working on any OS

**Comment by Klemek** [comment]
All PhantomJs tests are passing, the others failing are "Bad credentials" or "API rate limit exceeded", I really don't think this is because of my contribition nor this is the place to correct these failing tests.

**Pull request merged**